ds_judgements_public_ui/sass/includes/_footer.scss

```diff
@@ -62,23 +62,60 @@
     }
   }
 
+  &__ojl-paragraph {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__ojl-logo {
+    width: 50px;
+    height: 31px;
+    margin-right: $space-3;
+    margin-bottom: -$space-4;
```

Comment:
Negative margin suggests we're styling something unrelated to this to bring it closer to this element. We should apply the styling on whatever element is below this rather than here